iron-autogrow-textarea.html

```diff
@@ -202,6 +202,7 @@
 
     /**
      * Returns the underlying textarea.
+     * @type HTMLTextAreaElement
```

Comment:
@notwaldorf As far as I can see, this is the notation used for all getters in https://github.com/PolymerElements.

For example here:
https://github.com/PolymerElements/paper-header-panel/blob/master/paper-header-panel.html#L268

or here:
https://github.com/PolymerElements/iron-selector/blob/master/iron-selectable.html#L147